src/DappCard/DappCard.js

```diff
@@ -26,6 +26,9 @@ import { FormattedMessage } from 'react-intl';
 import DappVouchFor from './DappVouchFor';
 import styles from './DappCard.css';
 
+const DAPP_DAPP_VISIBLE_ID =
+  '0xa48bd8fd56c90c899135281967a6cf90865c221b46f27f9fbe2a236d74a64ea2'; // This dapp's id
```

Comment:
@jacogr I'm now hardcoding so that the dapp-dapp-visible cannot be hidden. Any ideas how I can get this id without hardcoding?

Reply:
It _should_ be fine - the idea is that it is static, since it makes it easier for us to manage in the registries.